src/test/resources/features/Registration.feature

```diff
@@ -16,20 +16,25 @@ Feature: Register for ECL
       | Scottish partnership          |
       | Sole trader                   |
 
-  Scenario: User registers a limited company supervised by an other professional body for AML that is liable for ECL
+  Scenario: User does not select whether or not they carry out AML-regulated activity between 1 April 2022 and 31 March 2023
     Given I am signed in to the registration journey
-    When I provide details of my limited company that is supervised by an other professional body and liable for ECL
-    Then I should be on the page that says Did you start AML-regulated activity in FY 2022?
+    When I do not select an option for whether or not I started AML regulated activity between 1 April 2022 and 31 March 2023
+    Then I should see an error that says Select an answer
 
-  Scenario: User's UK revenue is less than £10.2 million
+  Scenario: User selects no on whether or not they carry out AML-regulated activity between 1 April 2022 and 31 March 2023
     Given I am signed in to the registration journey
-    When I say that my UK revenue is less than £10.2 million
-    Then I should be on the page that says You do not need to register for the Economic Crime Levy
+    When I select No on whether or not I carried out AML-regulated activity between 1 April 2022 and 31 March 2023
+    Then I should see an error that says You do not need to register for the Economic Crime Levy
```

Comment:
```suggestion
    Then I should see a page that says You do not need to register for the Economic Crime Levy
```

it's not an error, it's a perfectly valid scenario

Reply:
correction done